manifests/m/MoheshwarAmarnathBiswas/UsefulToolsForWindows/1.1/MoheshwarAmarnathBiswas.UsefulToolsForWindows.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: MoheshwarAmarnathBiswas.UsefulToolsForWindows
+PackageVersion: 1.1

```

```suggestion
PackageVersion: 1.2
```
Version in control panel is 1.2

---

Oh no I messed up the installer itself. it should be 1.1. Anyway What should I do now? Should I rebuild the installer or just edit the package version?

---

I am doing rename cuz I am too lazy to rebuild installer.